05.ls/ls.rb

```diff
@@ -3,11 +3,15 @@
 COLUMN_LENGTH = 3
 
 def main
-  max_filename_char_length = count_filename_char
+  # mainメソッド内だけでも、どんな種類のファイルかを一目で分かりやすくしたいため、このように命名しました。
+  no_dot_files = load_files
 
-  prepare_files_to_display.each do |files|
+  max_filename_length = count_length_filename(no_dot_files)
```

Comment:
`count_length_filename`は`count_filename_length`の方が英語として自然かも。
ただ、`count_filename_length`だと「ファイル名の長さを数える」という意味になります。
ここでやっているのは「一番長いファイル名を見つけて返す」という処理なので、使う単語がちょっと違うような気がしますー。

まあ、もっと言ってしまえば、メソッド定義をなくしちゃっても、実は可読性はあまり落ちない（むしろ「一番長いファイル名を見つけて返す」をより適切に表現できている！？）、と考えることもできそうです。

```suggestion
  max_filename_length = no_dot_files.max_by(&:length).length
```